runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -199,11 +234,19 @@ private void buildBox(LootTrackerRecord record)
 			logsContainer.remove(box);
 			logsContainer.repaint();
 		});
-
-		// Create popup menu
-		final JPopupMenu popupMenu = new JPopupMenu();
-		popupMenu.setBorder(new EmptyBorder(5, 5, 5, 5));
 		popupMenu.add(reset);
+
+		if (groupLoot)
+		{
+			// Create details menu
+			final JMenuItem details = new JMenuItem("View details");
+			details.addActionListener(e ->
+			{
+				showDetailedView(box);
```

Comment:
Just set the detailedKey here, instead of exposing the ID from box (here you already have access to ID)